src/content/docs/en/guides/backend/xata.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Xata & Astro
+description: Add a serverless database with full-text search to your project with Xata
+type: backend
+service: Xata
+stub: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[Xata](https://xata.io) is a **Serverless Data Platform** because it combines the features of a relational database, a search engine, and an analytics engine by exposing a single consistent REST API.
+
+## Quick Intro
+
+Because Xata is a database, it’s ill-advised to connect directly to it from the frontend. There are 3 options to handle data-fetching from Xata in your Astro app. And understanding **when** fetch happens will help you identify the best strategy for your app:
+
+1. Fetching once at build-time and have static data.
+2. Using [Server-Side Rendering](/guides/server-side-rendering/) and fetching every time your page is rendered.
+3. Connecting to a backend API, e.g.: a Backend For Frontend (BFF), a Serverless Function, or an Edge Function.
+
+## Zero to Production
+
+Xata offers a starter template which is the fastest way to get a new project setup.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+npx degit xataio/examples/apps/starter-astro my-awesome-app
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+pnpx degit xataio/examples/apps/starter-astro my-awesome-app
```

Comment:
```suggestion
pnpm dlx degit xataio/examples/apps/starter-astro my-awesome-app
```

Reply:
Thanks!
I _think_ they're alias for the same thing, but I wasn't sure which one to use for consistency...

Reply:
Yup! They are in sense the same, overall in the docs we use `pnpm dlx`, since `pnpx` is depricated